src/legacy/core_plugins/kibana/public/table_list_view/_index.scss

```diff
@@ -0,0 +1,12 @@
+
+.kbnTableListView__actionIcon {
+  filter: grayscale(100%);
```

Comment:
https://caniuse.com/#search=grayscale

Did you test this to make sure it worked?

Reply:
It's a minor thing, so it doesn't need to work perfectly across browsers. I'm just trying to get the table to match the look / feel of the default EUI table which itself uses this filter.

The filter works in FF, Chrome, and Safari, though. Haven't tested in IE or Edge.

Reply:
Ideally `EuiInMemoryTable` would support to have different actions on different rows. I suggest we add a TODO here and open an issue in EUI to add this functionality - then we can clean it up here. What do you think @snide @chrisdavies ?